lib/bemhtml/runtime/index.js

```diff
@@ -389,6 +392,25 @@ BEMHTML.prototype.runOne = function runOne(json) {
   return res;
 };
 
+BEMHTML.prototype.tryRun = function tryRun(context, ent) {
+  var res;
+
+  try {
+    res = ent.run(context);
+  } catch (e) {
+    console.error('BEMXJST ERROR: cannot render ' +
```

Comment:
wouldn't it be more informative to show whole context:
```js
console.error('BEMXJST ERROR: cannot render ' + JSON.stringify(context, null, 4), e);
```

also it's possible to get rid of `res`:
```js
try {
    return ent.run(context);
} catch (e) {
    console.error(/* ... */);
}

return '';
```